src/commcare_cloud/ansible/requirements.yml

```diff
@@ -1,11 +1,14 @@
 ---
 roles:
-  - src: git+https://github.com/andrewrothstein/ansible-couchdb.git
-    version: fcb957ed038ab1c4fddcfef6b9c7617dcdeec9b7
-    name: andrewrothstein.couchdb
+#  - src: git+https://github.com/andrewrothstein/ansible-couchdb.git
+#    version: fcb957ed038ab1c4fddcfef6b9c7617dcdeec9b7
+#    name: andrewrothstein.couchdb
```

Comment:
We should just remove this rather than comment it out, right?

Reply:
Ah! No, we should actually keep it, I forgot to remove this change from the third commit.